templates/hobbygroups/index.html

```diff
@@ -41,7 +41,16 @@ <h1 id="resourcecenter-heading">Interessegrupper</h1>
         <div class="row" id="economic_support">
             <div class="col-xs-12 col-sm-12 col-md-12">
                 <div class="col-md-12">
-                    <h4>Hvordan søke om støtte til din interessegruppe?</h4>
+                    <h4>Hvordan opprette interessegruppe?</h4>
+                    <p>
+                        Dersom du har en ide til en interessegruppe sender du en mail til
+                        <a href="mailto:seniorkom@online.ntnu.no">seniorkom@online.ntnu.no</a> og melder interesse.<br>
+						Det trenger ikke være en lang mail, bare en kort forklaring på hva gruppen skal gjøre. Eneste kravet er at gruppen skal være åpen for alle Onlinere.
+                        Du kan lese mer hvordan opprette en interessegruppe
```

Comment:
sorry for nitpicking, but maybe "Du kan lese mer om hvordan man oppretter en interessegruppe her" or at least add  "om" before hvordan